src/components/Logout/index.jsx

```diff
@@ -0,0 +1,16 @@
+import { useNavigate } from 'react-router-dom';
```

Comment:
Would it make more sense to navigate to Login or to Landing Page? Since it might not be the same person loging in, but a new user subscribing?